### PR TITLE
CM-716 - Rename Coins to Tokens in all widgets

### PR DESCRIPTION
**Pull request by alejoloaiza**
### Hi👋, please prefix this PR's title with:

- [ ] `breaking-change:` if you have introduced modification that necessitates immediate adjustments by this SDK's users to their applications, clients, or integrations to avert disruptions to existing features or functionalities.
- [ ] `feat:`, `fix:`, `refactor:`, `docs:`, or `chore:`.

# Summary
Renaming from Coins to Tokens across multiple Widgets, see screenshots of local tests after the change
Screenshot 2024-06-12 at 2 54 39 PM
Screenshot 2024-06-12 at 2 40 17 PM
Screenshot 2024-06-12 at 2 34 49 PM
Screenshot 2024-06-12 at 2 29 55 PM
Screenshot 2024-06-12 at 2 10 44 PM

# Detail and impact of the change

## Security

# Anything else worth calling out?

### CODE REVIEW

**Review by mimi-imtbl — DISMISSED** [review]

**Review by jhesgodi — DISMISSED** [review]
🔥

File: packages/checkout/widgets-lib/src/locales/en.json [hunk 1]
```diff
@@ -234,6 +234,12 @@
         "title": "How would you like to add tokens?"
       },
       "topUpOptions": {
+        "onramp": {
+          "heading": "Buy tokens",
+          "caption": "With card, Google pay or Apple pay available.",
+          "subcaption": "Fees ",
+          "disabledCaption": ""
```
Comment (jhesgodi) [inline packages/checkout/widgets-lib/src/locales/en.json:241]:
```suggestion
          "heading": "Buy tokens"
```

Comment (jhesgodi) [reply packages/checkout/widgets-lib/src/locales/en.json:241]: these other options are not required, because are not being used

**Review by jhesgodi — CHANGES_REQUESTED** [review]

File: packages/checkout/widgets-lib/src/locales/en.json (hunk 1 above)
Comment (alejoloaiza) [reply packages/checkout/widgets-lib/src/locales/en.json]: updated

**Review by jhesgodi — APPROVED** [review]

**Pull request merged**